operator/config/crd/bases/customize.core.cnrm.cloud.google.com_controllerreconcilers.yaml

```diff
@@ -41,6 +41,20 @@ spec:
           spec:
             description: ControllerReconcilerSpec is the specification of ControllerReconciler.
             properties:
+              pprofConfig:
```

Comment:
Naming nits: it's all config, so I would suggest `pprofConfig` -> `pprof` or `profiling`

And then if we use `pprof`, `pprofPort` can just be `port`, and `pprofSupport` can just be `support` (or `supportedModes`, I originally thought pprofSupport was really `enabled`

If we do `profiling`, `pprofPort` and `pprofSupport` (or `pprofSupportedModes` or `pprofModes`) make sense, because then we don't have pprof duplicated (and we allow for other means of profiling in future)

Reply:
BTW I also wonder whether otel might be more useful than pprof.  pprof might be useful for tracking down memory usage, and might be useful for tracking down CPU usage, but I don't _think_ either of those are our biggest problems?  Not a blocker to adding pprof support first though :-)

Reply:
This is really last mile wiring. We already have pprof capability in the binaries themselves. We were just missing a way to turn it on. All for adding otel but this seemed like a quick win.

Reply:
Also this was inspired by the LRO resources where the controller went silent. Figured this would be useful for confirming that the controller was stack waiting on the LRO and not that something else was wrong.